demo/outbreak_forecast/requirements.txt

```diff
@@ -2,4 +2,5 @@ numpy
 matplotlib
 bokeh
 plotly
-altair
\ No newline at end of file
+altair
+opencv-python
```

Comment:
accident?

Reply:
The workflow file runs from main so I can't actually add a step to install open-cv until after merge. Will fix once 5.0 is released.